src/environment.cpp

```diff
@@ -1322,9 +1324,8 @@ void ServerEnvironment::step(float dtime)
 	/*
 		Manage active block list
 	*/
-	if(m_active_blocks_management_interval.step(dtime, 2.0))
-	{
-		ScopeProfiler sp(g_profiler, "SEnv: manage act. block list avg /2s", SPT_AVG);
+	if (m_active_blocks_management_interval.step(dtime, m_cache_abm_interval)) {
+		ScopeProfiler sp(g_profiler, "SEnv: manage act. block list avg /interval", SPT_AVG);
```

Comment:
`/2s` makes sense, but `/interval` doesn't make much sense since it isn't a number.  Just use `per interval`.

More importantly, this doesn't seem to actually be the ABM interval, but the management interval.  The ABM interval (below) was 1s but this was 2s.  Therefore, it seems like this will actually make ABMs coalesce *more*, by changing the ABM interval from 1s to 2s to match the management interval.  Is this correct?